includes/sanitizers/class-amp-blacklist-sanitizer.php

```diff
@@ -41,18 +48,28 @@ private function strip_attributes_recursive( $node, $bad_attributes, $bad_protoc
 				if ( 0 === stripos( $attribute_name, 'on' ) && $attribute_name != 'on' ) {
 					$node->removeAttribute( $attribute_name );
 					continue;
-				} elseif ( 'href' === $attribute_name ) {
+				} elseif ( 'href' === $attribute_name && 'a' !== $node_name ) {
```

Comment:
Only `<link>` and that's not valid in the body anyway. This is fine as-is in my opinion.